pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableInstances.java

```diff
@@ -165,9 +166,11 @@ public List<String> getLiveBrokersForTable(
   @ApiResponses(value = {
       @ApiResponse(code = 200, message = "Success"), @ApiResponse(code = 500, message = "Internal server error")
   })
-  public Map<String, List<InstanceInfo>> getLiveBrokers() {
+  public Map<String, List<InstanceInfo>> getLiveBrokers(
+          @ApiParam(value = "Table name (with or without type)", required = false)
+          @QueryParam("tablename") Optional<String> tablename) {
```

Comment:
Query param should be `tableName`
Shall we keep it consistent with other APIs where optional field is represented as `@Nullable`
```suggestion
          @QueryParam("tableName") @Nullable tableName) {
```